spec/controllers/api/v1/froggo_teams_controller_spec.rb

```diff
@@ -52,7 +52,7 @@
 
     context "when everything is ok" do
       before do
-        post :create, params: { name: "new_name", organization_id: platanus_org.id }, format: :json
+        post :create, params: { name: "name", organization_id: platanus_org.id, new_members_ids: [user.id] }, format: :json
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Layout/LineLength: Line is too long. [123/100]